img/star.svg

```diff
@@ -0,0 +1,5 @@
+<svg alt="Star" focusable="false" enable-background="new 0 0 16 16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
set a default height and width so in case the css is not loaded we have something not so bad